src/DB_Command.php

```diff
@@ -456,6 +456,177 @@ public function tables( $args, $assoc_args ) {
 		}
 	}
 
+	/**
+	 * Display the database name and size as ASCII table.
```

Comment:
Because you can also display the database name and size in other formats too, does it make sense to explicitly describe this as "ASCII table?